src/utils/mailup.ts

```diff
@@ -293,21 +293,20 @@ export function MailUpTransport(
 
       const email = errorOrEmail.value;
 
-      const errorOrResponse = await sendTransactionalMail(
-        options.creds,
-        email,
-        fetchAgent
-      ).run();
-
-      if (isRight(errorOrResponse)) {
-        // tslint:disable-next-line:no-null-keyword
-        return callback(null, {
-          ...errorOrResponse.value,
-          messageId: mail.data.messageId
-        });
-      } else {
-        return callback(errorOrResponse.value, undefined);
-      }
+      sendTransactionalMail(options.creds, email, fetchAgent)
```

Comment:
do we need a `return` here ?

Reply:
nope, [the caller](https://github.com/nodemailer/nodemailer/blob/7750e048ff79d099500061497fb45b2861a7652c/lib/mailer/index.js#L214) doesn't expect a return value